web-components/src/[sandbox]/sandbox.mock.ts

```diff
@@ -12,6 +12,11 @@ export const comboBoxOptions = [
   "Argentina"
 ];
 
+export const tableErrors = [
+  "Ctrl + Alt + C",
```

Comment:
there might be a cell with this content, that is not an error and does not need to e marked. You need to choose teh location instead of content

Reply:
Fixed